src/components/PuzzleModal/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react"
+import "./index.scss"
+import { ReactComponent as ExitIcon } from "../../images/icons/circle-xmark-solid.svg"
+
+interface PuzzleProps {
+  children: React.ReactNode
+  question: string
+  answer: string
+}
+
+const link =
+  "https://docs.google.com/forms/d/e/1FAIpQLSfyGGDipFozDbQrmDBIoGvCwLcn5Lec3GMNnCgsRWKp9Q4syA/viewform"
+
+const PuzzleModal: React.FC<PuzzleProps> = ({
+  children,
+  question,
+  answer,
+}: PuzzleProps) => {
+  const [response, setResponse] = useState("")
+  const [success, setSuccess] = useState(false)
+  const [errorMessage, setErrorMessage] = useState("")
+  const [isPuzzleHidden, setIsPuzzleHidden] = useState(true)
+  const handleExit = () => setIsPuzzleHidden(true)
+  const hiddenOrNot = isPuzzleHidden
+    ? "puzzlemodal__container hidden"
+    : "puzzlemodal__container displayed"
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setResponse(event.target.value)
+  }
+
+  const handleSubmit = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.key === "Enter") {
+      if (response.toLowerCase() === answer) {
+        setSuccess(true)
+        setErrorMessage("")
+      } else {
+        setErrorMessage("Sorry, that's incorrect. Guess again")
+      }
+    }
+  }
+  const SuccessMessage = (
+    <div className='successMessage'>
```

Comment:
<img width="1116" alt="pop-up-puzzle-sizing" src="https://user-images.githubusercontent.com/38510322/147322781-dd55ab55-a690-4442-a732-263ebf1e61e5.PNG">

The sizing still seems to be a little awkard for the success message.